mlxtend/feature_selection/exhaustive_feature_selector.py

```diff
@@ -212,11 +261,22 @@ def fit(self, X, y, custom_feature_names=None, groups=None, **fit_params):
         self.best_feature_names_ = None
         self.best_score_ = None
 
+        self.fixed_features = self.fixed_features
```

Comment:
I think this line (264) does nothing. Did you mean to write something else here?